chainntnfs/bitcoindnotify/bitcoind_test.go

```diff
@@ -109,11 +109,15 @@ func syncNotifierWithMiner(t *testing.T, notifier *BitcoindNotifier,
 // TestHistoricalConfDetailsTxIndex ensures that we correctly retrieve
 // historical confirmation details using the backend node's txindex.
 func TestHistoricalConfDetailsTxIndex(t *testing.T) {
-	t.Run("rpc polling enabled", func(st *testing.T) {
+	success := t.Run("rpc polling enabled", func(st *testing.T) {
 		st.Parallel()
 		testHistoricalConfDetailsTxIndex(st, true)
 	})
 
+	if !success {
```

Comment:
But normally the above test should always succeed or ?

Reply:
Yes, this is just a shortcut to avoid running all other test cases if one fails.